cli/src/commands/extensions/mod.rs

```diff
@@ -73,8 +73,14 @@ pub async fn handle_extensions(matches: &ArgMatches) -> CommandResult {
 
 /// Handle the `extension add` subcommand path.
 /// Add the extension from the specified path.
-async fn handle_add_extension(path: PathBuf) -> CommandResult {
-    let extension = Extension::try_from(path)?;
+async fn handle_add_extension(path: &str) -> CommandResult {
+    // NOTE: Extension installation without slashes is reserved for the marketplace.
+    if !path.contains('/') {
```

Comment:
Are we supporting windows? If so, wouldn't we need to check for backslashes as well?